kit/src/elements/file/mod.rs

```diff
@@ -28,6 +30,13 @@ pub fn get_text(cx: &Scope<Props>) -> String {
     }
 }
 
+pub fn get_aria_label(cx: &Scope<Props>) -> String {
+    match &cx.props.aria_label {
```

Comment:
As a suggestion (no need to accept it here)
```suggestion
    cx.props.aria_label.clone().unwrap_or_default()
```
That way you dont have to worry about match conditions (which can apply to similar functions like this)

Reply:
Thanks! @dariusc93 I went ahead and changed the get_aria_label functions that I have added before to work like this